tigerpath/majors_and_certificates/majors/CLA_2018.json

```diff
@@ -0,0 +1,518 @@
+            },
+            {
+              "name": "Ancient Greek or Latin",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "Each student must successfully complete either Ancient Greek or Latin to the level of 108 or achieve an equivalent level of knowledge, as demonstrated through test scores (SAT, AP), a placement exam, or coursework (including through a summer language program). However, at least one language course must be taken at Princeton.",
+              "course_list": [
+                "LAT 108",
+                "CLG 108"
+              ]
+            },
+            {
+              "name": "Departmentals",
+              "max_counted": 1,
+              "min_needed": 8,
+              "explanation": "Eight departmental courses are required. These must include one survey course on ancient Greek history (CLA 216, CLA 217) and one survey course on Roman history (CLA 218, CLA 219); two seminars at the 300 level focusing on ancient history (e.g., CLA 326, CLA 327: Topics in Ancient History; CLA 324: Classical Historians and their Philosophies of History; CLA 325: Roman Law); one course on material culture (e.g., ART 202: Greek Art; ART308: Roman Cities and Countryside); two courses focusing in whole or in part on classical civilization (see list below); and one history course on a non-classical pre-nineteenth century civilization (e.g., EAS 335: Early Chinese History to 221; NES 220: Jews, Muslims, and Christians in the Middle Ages; HIS 350 History of France, 1685-1800). Students must also take the Junior Seminar in the fall of their junior year.",
+              "req_list": [
+                {
+                  "name": "Ancient Greek History",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "These must include one survey course on ancient Greek history (CLA 216, CLA 217).",
+                  "course_list": [
+                    "CLA 216",
+                    "CLA 217"
+                  ]
+                },
+                {
+                  "name": "Roman History",
+                  "max_counted": 1,
+                  "min_needed": 1,
+                  "explanation": "One survey course on Roman history (CLA 218, CLA 219)",
+                  "course_list": [
+                    "CLA 218",
+                    "CLA 219"
+                  ]
+                },
+                {
+                  "name": "Ancient History Seminars",
+                  "max_counted": 2,
+                  "min_needed": 2,
+                  "explanation": "<i>Editor's Note: Some of these courses fall under this requirement but TigerPath does not verify this.</i>\n\nTwo seminars at the 300 level focusing on ancient history (e.g., CLA 326, CLA 327: Topics in Ancient History; CLA 324: Classical Historians and their Philosophies of History; CLA 325: Roman Law)",
+                  "course_list": [
+                    "ART 300",

```

Comment:
Where does this course list come from again? Is it just all courses that could possibly count as a departmental?

Reply:
https://classics.princeton.edu/programs/undergraduate/tracks/ancient-history-track